projects/packages/forms/src/dashboard/inbox/export-responses/style.scss

```diff
@@ -59,3 +59,9 @@
 .jp-forms__export-modal-card-body-cta {
 	align-self: flex-end;
 }
+
+.jp-forms__export-button--large-green.components-button.has-icon.has-text {
```

Comment:
Do we need to include the core component classes like `components-button`?

Reply:
`button.jp-forms__export-button--large-green` might work, or another way (albeit a bit of a hack) is `.jp-forms__export-button--large-green.jp-forms__export-button--large-green`

Consider core component classes as instable API so they could change without us noticing.

Reply:
`button.jp-forms__export-button--large-green` is not sufficient. But `.jp-forms__export-button--large-green.jp-forms__export-button--large-green` worked. With this updated, the button also needed some extra styles on hover to again override some core styling. So the resulting selectors are now:

```
.jp-forms__export-button--large-green.jp-forms__export-button--large-green,
.jp-forms__export-button--large-green.jp-forms__export-button--large-green:hover:not(:disabled,[aria-disabled="true"],.is-pressed)
```

Reply:
👍  Fine for this PR but let's check with @ilonagl on visuals for the button so that we could move to use just core button instead of needing anything extra here.